troposphere/static/js/components/settings/AdvancedSettingsPage.jsx

```diff
@@ -1,7 +1,11 @@
 import React from "react";
+
 import SSHConfiguration from "components/settings/advanced/SSHConfiguration";
+import ClientCredentials from "components/settings/advanced/ClientCredentials";
```

Comment:
This is confusing.  The actual component is named "ClientCredentials" and it launches a modal to do the "export".